datamonster_api/tests/lib/regression.py

```diff
@@ -109,15 +191,13 @@ def test_data_source():
         ds.get_data(company, Aggregation(period="quarter", company=company))
 
 
-def test_data_source_2():
-    """ Test FactSet Actuals Sales Quarterly, one of the more popular data sets
+def test_get_data_factset():
+    """ Test `FactSet Actuals Sales Quarterly` one of the more popular data sets
     """
     company = dm.get_company_by_id(335)
-
     sales = next(dm.get_datasources(query="FactSet Actuals Sales Quarterly"))
     assert sales.id == "bdcac6ae-4f31-4aaf-a92a-12854f09c768"
```

Comment:
Should these UUIDs be saved off as constants somewhere?

Reply:
ok